spec/dispatch_spec.rb

```diff
@@ -18,15 +18,12 @@ module QueueBus
       dispatch.execute("my_event", {"bus_event_type" => "my_event", "ok" => true})
       Runner1.value.should == 1
       Runner1.attributes.should == {"bus_event_type" => "my_event", "ok" => true}
-      
     end
-    
+
     it "should not crash if not there" do
-      lambda {
-        Dispatch.new("d").execute("fdkjh", "bus_event_type" => "fdkjh")
-      }.should_not raise_error
+      Dispatch.new("d").execute("fdkjh", "bus_event_type" => "fdkjh").should == nil
```

Comment:
Not it wasn't failing, but rspec gives a warning - they don't recommend using `should_not raise_error`. One should assert on what is expected, instead of what is not happening ;)